docs/source/workflow_inputs.rst

```diff
@@ -820,17 +820,18 @@ If insulation layers are provided, additional information is entered in each ``F
 
   .. [#] When NominalRValue is non-zero, DistanceToBottomOfInsulation must be greater than DistanceToTopOfInsulation and less than or equal to FoundationWall/Height.
 
-HPXML Frame Floors
+HPXML Floors
 ******************
 
-Each floor/ceiling surface that is not in contact with the ground (Slab) nor adjacent to ambient conditions above (Roof) is entered as an ``/HPXML/Building/BuildingDetails/Enclosure/FrameFloors/FrameFloor``.
+Each floor/ceiling surface that is not in contact with the ground (Slab) nor adjacent to ambient conditions above (Roof) is entered as an ``/HPXML/Building/BuildingDetails/Enclosure/Floors/Floor``.
 
   ======================================  ========  ============  ===========  ========  ========  ============================
   Element                                 Type      Units         Constraints  Required  Default   Notes
   ======================================  ========  ============  ===========  ========  ========  ============================
   ``SystemIdentifier``                    id                                   Yes                 Unique identifier
   ``ExteriorAdjacentTo``                  string                  See [#]_     Yes                 Exterior adjacent space type
   ``InteriorAdjacentTo``                  string                  See [#]_     Yes                 Interior adjacent space type
+  ``FloorType``                           element                 1 [#]_       Yes                 Floor type
```

Comment:
I'm confused. Are you intending to allow this input as part of this PR or not? You have added it to the documentation, but it's not a new input in EPvalidator.xml nor is it used by the workflow to adjust how the floor construction gets defined. So it doesn't do anything.

I would be okay with tackling this in two PRs. In this PR, just to handle the FrameFloor -> Floor change (i.e., revert documentation changes about FloorType). And then once that is merged, you can open a new PR to make all changes related to a new FloorType input.